### ISSUE TITLE
[A11y] 1.4.3 Contrast (Minimum) - Copy Has Insufficient Contrast Against Background

### ISSUE THREAD

**Issue by dalonsoa** [research-software-directory/RSD-as-a-service#1769]
See https://github.com/ImperialCollegeLondon/RSD-as-a-service/issues/37 for details.

This is on us, but I add it here for completeness as it might affect the pages of other people.

**Comment by dalonsoa** [comment]
Mostly sorted, but still some areas of improvement. Will work on them over the coming weeks. It has minor impact on accessibility, it would seem, so it should be OK.
